include/z64save.h

```diff
@@ -162,17 +205,17 @@ typedef struct {
     /* 0x13C2 */ char unk_13C2[0x0001];
     /* 0x13C3 */ u8 retainWeatherMode;
     /* 0x13C4 */ s16 dogParams;
-    /* 0x13C6 */ u8 textTriggerFlags;
+    /* 0x13C6 */ u8 envHazardTextTriggerFlags;
     /* 0x13C7 */ u8 showTitleCard;
     /* 0x13C8 */ s16 nayrusLoveTimer;
     /* 0x13CA */ char unk_13CA[0x0002];
     /* 0x13CC */ s16 rupeeAccumulator;
-    /* 0x13CE */ s16 timer1State;
-    /* 0x13D0 */ s16 timer1Value;
-    /* 0x13D2 */ s16 timer2State;
-    /* 0x13D4 */ s16 timer2Value;
-    /* 0x13D6 */ s16 timerX[2];
-    /* 0x13DA */ s16 timerY[2];
+    /* 0x13CE */ s16 timerState; // See `TimerState`
+    /* 0x13D0 */ s16 timerTime; // in seconds
+    /* 0x13D2 */ s16 subTimerState; // See `SubTimerState`
+    /* 0x13D4 */ s16 subTimerTime; // in seconds
```

Comment:
will admit "timer time" feels a bit redundant to me. is there a specific reason you wanted to change it from value like it was? think it works fine

Reply:
`Value` to me felt a bit to vague and general, so replacing it with the redundant `Time` did feel slightly better to me since it at least indicates that it's the "time" instead of the many other quantities associated with the timer.

`TimerTime` doesn't flow the greatest though, so I'm not strongly opposed to going back to `TimerValue` or finding something else for the actual time left on the timer.

Reply:
In no particular order
`timerValue`
`timerTime`
`timerSeconds`
`timerCount`

I myself don't have a preference

Reply:
`timerSeconds` works for me